lightning_examples/finetuning-scheduler/.meta.yml

```
@@ -1,11 +1,11 @@
 title: Finetuning Scheduler
 author: "[Dan Dale](https://github.com/speediedan)"
 created: 2021-11-29
-updated: 2022-05-10
+updated: 2022-06-10
 license: CC BY-SA
-build: 3
+build: 4
```

```suggestion
build: 0
```
I know it is not described; this is just to trigger build is if there are no other changes in config nor script